src/pages/PoolDetails/index.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { usePoolData } from 'graphql/thegraph/PoolData'
+import Router from 'react-router-dom'
+import { mocked } from 'test-utils/mocked'
+import { render, screen, waitFor } from 'test-utils/render'
+
+import PoolDetails from '.'
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(),
+}))
+
+jest.mock('graphql/thegraph/PoolData', () => {
+  const originalModule = jest.requireActual('graphql/thegraph/PoolData')
+  return {
+    ...originalModule,
+    usePoolData: jest.fn(),
+  }
+})
+
+const validParams = { poolAddress: '0x88e6a0c2ddd26feeb64f039a2c41296fcb3f5640', chainName: 'ethereum' }
+const validPoolDataResponse = {
+  data: {
+    __typename: 'Pool' as const,
+    id: '0x88e6a0c2ddd26feeb64f039a2c41296fcb3f5640',
+    feeTier: '500',
+    liquidity: '32118065613640312417',
+    sqrtPrice: '1943494374075311739809880994923792',
+    tick: '202163',
+    token0: {
+      __typename: 'Token' as const,
+      id: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+      symbol: 'USDC',
+      name: 'USD Coin',
+      decimals: '6',
+      derivedETH: '0.000602062055419695968472438533210813',
+    },
+    token1: {
+      __typename: 'Token' as const,
+      id: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+      symbol: 'WETH',
+      name: 'Wrapped Ether',
+      decimals: '18',
+      derivedETH: '1',
+    },
+    token0Price: '1661.85294822715829371652214854595',
+    token1Price: '0.0006017379582632664031212782038199158',
+    volumeUSD: '394920157156.0515346899898790592366',
+    volumeToken0: '394894081779.781168',
+    volumeToken1: '190965971.266407832255075308',
+    txCount: '5406827',
+    totalValueLockedToken0: '180078648.881221',
+    totalValueLockedToken1: '142782.017882048454494774',
+    totalValueLockedUSD: '417233634.1468435997761171520463339',
+  },
+  loading: false,
+}
```

Comment:
consider starting a mock data directory, since this is a pattern we'll probably have to replicate